rasa/shared/utils/tensorflow/tf_to_numpy.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, Optional
+from tensorflow import Tensor
+
+
+def values_to_numpy(data: Optional[Dict[Any, Any]]) -> Optional[Dict[Any, Any]]:
```

Comment:
mhm, I think we should keep ML things as much as possible out of `rasa.shared`. Also in the interest of the research team: The less your code affects downstream dependencies (such as Rasa X), the more flexible are you in changing your code without having to consider effects on downstream dependencies such as Rasa X. Especially in your case you can simply call `values_to_numpy` before calling `add_diagnostic_data`

Reply:
I missed that one, it should be in `rasa/utils/tensorflow/tf_to_numpy.py`

Reply:
Ok, I moved it back there.